### ISSUE TITLE
Autocreate namespace if not present

### ISSUE THREAD

**Issue by QwertyQaz414** [devspace-cloud/devspace#1127]
**Is your feature request related to a problem?**  
Yes

**Which solution do you suggest?**  
One of the following:
- Add an extra optional flag like `ensureNamespace: true` to enable this.
- Add a global config/env var to enable this for all deployment
- Change the default behaviour to automatically do this

IMHO the first option is nicer.

**Which alternative solutions exist?**  
hooks but are very verbose and overkill for a task that needs to be executed only once.

**Additional context**  
in my case i have the following (which is part of a bigger file):
```yaml
version: v1beta9
deployments:
- name: pgadmin
  namespace: management
  helm:
    chart:
      name: pgadmin4
      repo: https://helm.runix.net/
      version: 1.2.25
```

however, when somebody runs it for the first time devspace aborts because no namespace `management` exists.

This is not really a problem for Kustomization/manifests since it can be added inside, but for helm charts deployments the only solutions I found are:
1. add an hook which calls a yaml with the namespace definition inside before all deployments. (cannot simply use `kubectl create ns management` because it's not idempotent)
2. have a separate deployment only for setup stuff.

I personally don't find neither solutions very appealing nor following k8s style.

Any suggestions?

/kind feature

**Comment by FabianKramm** [comment]
@QwertyQaz414 thanks for reporting this issue! Makes sense, I think we can add this feature!

**Comment by FabianKramm** [comment]
@QwertyQaz414 implemented in v4.14.0

**Comment by QwertyQaz414** [comment]
Cool 👍
